packages/core/bootstrap/src/lib/ws/reducer.ts

```diff
@@ -209,6 +203,7 @@ export const subscriptionsReducer = createReducer<SubscriptionsState>(
 
       state.all[key].active = false
       state.all[key].unsubscribed = true
+      state.all[key].subscribing = 0
```

Comment:
Want to double check this.  Should this be set to 0 if the adapter successfully unsubscribes?